media-sound/ario/ario-1.6-r1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg-utils
+
+DESCRIPTION="GTK client for MPD inspired by Rhythmbox but much lighter and faster"
+HOMEPAGE="http://ario-player.sourceforge.net"
+SRC_URI="mirror://sourceforge/${PN}-player/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dbus debug +idle nls taglib zeroconf"
```

Comment:
Compared to 1.5.1-r1 libnotify, python and scrobbler USE flags are gone. Why? Does upstream support them no longer? Were any new features added?

Reply:
It is explained in the commit message ;-)